services/mcp-test-server/.env.example

```
@@ -0,0 +1,42 @@
+# --- Keycloak / OAuth ---
+KEYCLOAK_REALM=local
+# Update KEYCLOAK_REALM and OIDC_ISSUER to match your deployment
+OIDC_ISSUER=http://127.0.0.1:5050/auth/realms/local
+OIDC_AUDIENCE=http://127.0.0.1:8770/mcp
+# --- MCP Service Metadata ---
+MCP_TEST_SERVICE_NAME=mcp-test-server
+MCP_TEST_SERVICE_NAME_HUMAN=MCP Test Server
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The MCP_TEST_SERVICE_NAME_HUMAN key should go before the OIDC_AUDIENCE key

(UnorderedKey)